packages/core/README.md

```diff
@@ -26,32 +26,4 @@ pnpm i @alchemy/aa-core
 
 You can create a provider like so:
 
-```typescript
-import {
-  LightSmartContractAccount,
-  getDefaultLightAccountFactoryAddress,
-} from "@alchemy/aa-accounts";
-import {
-  LocalAccountSigner,
-  SmartAccountProvider,
-  SmartAccountSigner,
-} from "@alchemy/aa-core";
-import { polygonMumbai } from "@alchemy/aa-core";
-
-const chain = polygonMumbai;
-const owner: SmartAccountSigner =
-  LocalAccountSigner.mnemonicToAccountSigner(YOUR_OWNER_MNEMONIC);
-
-export const provider = new SmartAccountProvider({
-  rpcProvider: "https://polygon-mumbai.g.alchemy.com/v2/demo",
-  chain,
-}).connect(
-  (rpcClient) =>
-    new LightSmartContractAccount({
-      chain,
-      factoryAddress: getDefaultLightAccountFactoryAddress(chain),
-      rpcClient,
-      owner,
-    })
-);
-```
+<<< @/snippets/aa-alchemy/smartAccountClient.ts [smartAccountClient.ts]
```

Comment:
I'm not sure this works for the README actually since this is just for `vitepress`

Reply:
ah gotcha. will revert